AsyncDisplayKit/ASCollectionView.h

```diff
@@ -171,6 +171,13 @@ NS_ASSUME_NONNULL_BEGIN
 - (void)relayoutItems;
 
 /**
+ * Triggers a recalulation of leadingScreensForBatching
+ * @discussion During various layout changes that trigger content size changes, batch fetching needs to recalulate it's number of screens left to scroll before the delegate -collectionView:beginBatchFetchingWithContext: is called. Call when a layout transition is complete, or it will have no effect.
+ *
+ */
+- (void)invalidateBatchFetchMeasurements;
```

Comment:
Hmm, I guess I am not familiar enough with the use case.  Is there no other call the ASCollectionView can intercept from the UICollectionViewDelegate?  Or from the layout?  There is a proxy so that ASCollectionView is able to listen to absolutely any kind of delegate / datasource call...